independent-publisher-connectors/Bitly/apiDefinition.swagger.json

```diff
@@ -0,0 +1,10627 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Bitly",

```

Comment:
Hi @troystaylor, it looks like we have another [Bitly ](https://docs.microsoft.com/en-us/connectors/bitly/)connector. can you please change the name if the connector, maybe to "BitlyIP".
Thank you very much